src/core/scene/a-scene.js

```diff
@@ -320,7 +364,16 @@ module.exports = registerElement('a-scene', {
         if (this.isPlaying) {
           this.tick(time, timeDelta);
         }
-        this.effect.render(this.object3D, camera);
+        // Postproc must be explicitly enabled by a component or system.
```

Comment:
I will take out the example from this PR, the design is pretty outdated. I'll just leave the logic and put examples in a different repo for inspection